src/lava/lib/peripherals/liveplot/liveplot.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import cv2
+import numpy as np
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import InPort
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.ports import PyInPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.decorator import implements, requires
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+
+import time
+
+
+class OpenCVDisplay(AbstractProcess):
+    """Process that receives datas from connected processmodel.
+    The frame data which are needed to display from connected processmodels
+    need a corresponding InPort which is connected to an OutPort.
+    The number of input ports is decided by the number of
+    process models connected to this opencv process model.
+    """
```

Comment:
This docstring does not adhere to the docstring format specified in Lava's developer guidelines (see https://lava-nc.org/developer_guide.html#docstring-format). Format has to be : 
```python
def function(self, arg1, arg2):
""" <Short description>
<Optional: Detailed description>

Parameters
----------
arg1
arg2
"""
```

An alternative would be : 
```python
""" Process that display incoming image with OpenCV.

This Process continuously displays images coming through its InPort in an OpenCV window.

Parameters
----------
shape: tuple
    Shape of the InPort.
"""
```

Depending on whether this is usable with Grayscale images, RGB images, or both, a mention of this should also be added in the docstring (preferably in the "Detailed description" section).

Reply:
Edited the docstring